dicoogle/src/main/resources/webapp/js/components/indexer/IndexStatusView.js

```diff
@@ -1,20 +1,37 @@
 import React from 'react';
 import {IndexStatusActions} from "../../actions/indexStatusAction";
 import {IndexStatusStore} from "../../stores/indexStatusStore";
+import {PluginActions} from "../../actions/pluginActions";
```

Comment:
With the changes in pluginActions.js, you would now do `import * as PluginActions from "../../actions/pluginActions";`